examples/google-tables/pack.ts

```diff
@@ -0,0 +1,184 @@
+import type {RowsContinuation} from "./types";
+import * as coda from "@codahq/packs-sdk";
+import {formatRowForApi} from "./helpers";
+import {formatRowForSchema} from "./helpers";
+import {getPropertySchema} from "./helpers";
+import {getRows} from "./helpers";
+import {getTable} from "./helpers";
+import {getTableUrl} from "./helpers";
+import {getTables} from "./helpers";
+import {updateRow} from "./helpers";
+
+export const pack = coda.newPack();
+
+// A base schema, extended with additional properties depending on the table.
+export const BaseRowSchema = coda.makeObjectSchema({
+  properties: {
+    rowId: {
+      description: "Internal ID of the row.",
+      type: coda.ValueType.String,
+      fromKey: "name",
+      required: true,
+    },
+    rowLabel: {
+      type: coda.ValueType.String,
+      required: true,
+    },
+    lastUpdated: {
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.DateTime,
+      fromKey: "updateTime",
+    },
+  },
+  displayProperty: "rowLabel",
+  idProperty: "rowId",
+  featuredProperties: [],
+});
+
+pack.addDynamicSyncTable({
+  name: "Table",
+  description: "Syncs records from a Google Tables table.",
+  identityName: "Table",
+  placeholderSchema: BaseRowSchema,
+  listDynamicUrls: async function (context) {
```

Comment:
Implement `searchDynamicUrls` as well?

Reply:
The API doesn't have a search capability, but I can implement a simple one with filtering. Done.